classes/file_storage.php

```diff
@@ -544,6 +544,7 @@ private static function readFileTree($source, $options, $archive = null, $relati
         if (empty($archive) && $exportzip) {
             $archive = new \ZipArchive();
             $path = tempnam(get_request_storage_directory(),'libdir');
+            unlink($path);
```

Comment:
instead of unlinking, it looks like the line below is buggy, and should be | instead of || to do the bitwise OR for the flag. That might just supress the message.

Reply:
You're right it did turn out to be an issue with that, that's corrected now